<ui version="4.0" >
 <class>EmoticonsOptionsClass</class>
 <widget class="QWidget" name="EmoticonsOptionsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>234</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="lwtEmoticons" />
   </item>
   <item>
    <widget class="QWidget" native="1" name="wdtButtons" >
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="tbtUp" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbtDown" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
